.ruby-version

```diff
@@ -1 +1 @@
-2.3.0
+2.3.3
```

Comment:
Those changes are not actually related to this PR right? Something went wrong when rebasing?

Reply:
I rebased on master 😔 . Don't know why it included that change on my PR 🤔 . I can try to rebase again.

Reply:
Nah, I guess it's fine... happens to me also a lot